resources/sass/core/_dark_header.scss

```diff
@@ -52,6 +50,15 @@ body.theme-dark {
           }
 
           .media {
+            > a:first-of-type {
```

Comment:
Sorry, but that is completely ambiguous to me. Looking at this style I have no idea what `a:first-of-type` and `> div` really are.

Could you add appropriate class names to these elements, please?

Reply:
@danon Done